crates/ra_assists/src/assists/early_return.rs

```diff
@@ -37,7 +37,9 @@ use crate::{
 // ```
 pub(crate) fn convert_to_guarded_return(ctx: AssistCtx<impl HirDatabase>) -> Option<Assist> {
     let if_expr: ast::IfExpr = ctx.find_node_at_offset()?;
-    let expr = if_expr.condition()?.expr()?;
+    let cond = if_expr.condition()?;
+    let pat = &cond.pat();
```

Comment:
I think this variable is not really necesasry. `cond.pat` is more readable than `pat` at the call-site